lib/plugins/pluginUtils.js

```diff
@@ -173,3 +173,5 @@ exports.createProcessAttribute = createProcessAttribute;
 exports.createUpdateFilter = createUpdateFilter;
 exports.createQueryFilter = createQueryFilter;
 exports.identityFilter = identityFilter;
+exports.createNgsi2Entity = createNgsi2Entity;
```

Comment:
Following the philosophy used by PR #1, this PR should include this in the CNR file:

```
- Add: NGSIv2 support for Timestamp Processing plugin, through configuration (#527)
```

Reply:
Fixed in 50dec0c8461814b2cd32423764049e738f156571